mpisppy/cylinders/xhatxbar_bounder.py

```diff
@@ -58,11 +58,7 @@ def ib_prep(self):
 
         self.opt._update_E1()  
         if (abs(1 - self.opt.E1) > self.opt.E1_tolerance):
-            if self.opt.cylinder_rank == 0:
-                print("ERROR")
-                print("Total probability of scenarios was ", self.opt.E1)
-                print("E1_tolerance = ", self.opt.E1_tolerance)
-            quit()
+            raise RuntimeError(f"Total probability of scenarios was {self.E1};  E1_tolerance = ", self.E1_tolerance)
```

Comment:
Same comment I raised elsewhere - why can is the cylinder_rank=0 check dropped?

Reply:
Same reply: It is not clear to me whether the rank 0 check (which was my idea in this case) was a good idea. I think you want every rank to shut down if infeasibility is detected rather than relying on the user to have done -m mpy4py so it will shut everyone down